tsdb/shard.go

```diff
@@ -452,7 +452,16 @@ func (s *Shard) IsIdle() bool {
 	if err != nil {
 		return true
 	}
-	return engine.IsIdle()
+	return engine.IsIdle(false)
+}
+
+// IsIdle return true if the shard is not receiving writes and is fully compacted.
```

Comment:
Godoc needs updating

Reply:
Why make 2 separate methods here instead of adding the `bool` parameter like everywhere else?

Reply:
Good point.